lib/gooddata/models/model.rb

```diff
@@ -86,7 +86,7 @@ module Model
       'GDC.time.day_us_noleading', # M/d/yy
     ]
 
-    GD_DATA_TYPES = ['BIGINT', 'DOUBLE', 'INTEGER', 'INT', /^VARCHAR\(\d{1,4}\)$/i, /^DECIMAL\(\d{1,3},\s*\d{1,3}\)$/i]
+    GD_DATA_TYPES = ['BIGINT', 'DOUBLE', 'INTEGER', 'INT', /^VARCHAR\(\d{1,4}\)$/i, 'VARCHAR(10000)', /^DECIMAL\(\d{1,3},\s*\d{1,3}\)$/i]
```

Comment:
This is the place I put the regression test: https://github.com/gooddata/gooddata-ruby/pull/971/files#diff-e4bed3a5d5113ec31ced0246278b6145R33